src/features/Configuration/Configuration.tsx

```diff
@@ -52,101 +54,98 @@ const BoardController: React.FC<{ active: boolean }> = ({ active }) => {
     const boardRevision = useSelector(getBoardRevisionSemver);
 
     if (device) {
-        console.log('Got device %o %s', device, boardRevision);
-        console.log(
-            'Device with boardVersion: %s',
-            device?.devkit?.boardVersion
+        logger.debug(
+            `Got device ${JSON.stringify(device)} ${JSON.stringify(
+                boardRevision
+            )}`
+        );
+        logger.debug(
+            `Device with boardVersion: ${JSON.stringify(
+                device?.devkit?.boardVersion
+            )}`
         );
```

Comment:
If the plan is to keep this, we could consider extracting the logging into a function?

```typescript
// Define it somewhere:
const logDeviceInfo = (device: Device) => {
     logger.debug(
            `Got device ${JSON.stringify(device)} ${JSON.stringify(
                boardRevision
            )}`
        );
        logger.debug(
            `Device with boardVersion: ${JSON.stringify(
                device?.devkit?.boardVersion
            )}
}

// ... and call it
logDeviceInfo(device);
```
sorry for the formatting, wish Github had autoformatting